config/crd/bases/self-node-remediation.medik8s.io_selfnoderemediations.yaml

```diff
@@ -82,9 +82,10 @@ spec:
                     description: Spec defines the behavior of a node. https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
                     properties:
                       configSource:
-                        description: If specified, the source to get node configuration
-                          from The DynamicKubeletConfig feature gate must be enabled
-                          for the Kubelet to use this field
+                        description: 'Deprecated: Previously used to specify the source
+                          of the node''s configuration for the DynamicKubeletConfig
```

Comment:
NIT: Why double apostrophe(`''`)? 
I assume it was supposed to be `node's` rather than `node''s`

Reply:
We are not responsible for the text change, we just see it because of the core updates